<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;黑体&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>欢迎您，</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 75 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>您的队伍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>您的ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>您的积分：</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>测试赛</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>第一关：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B1Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关卡介绍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="U1Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="F1Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="U2Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>第二关：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B2Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关卡介绍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="F2Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="U3Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>第三关：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B3Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关卡介绍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="F3Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="U4Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>第四关：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B4Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关卡介绍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="F4Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="U5Button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>第五关：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B5Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关卡介绍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="F5Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QLabel" name="S1Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="S2Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="S3Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="S4Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="S5Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="RankingButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查看排行榜</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>排位赛</string>
    </attribute>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>选择地图：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="MapCombo">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>选择对手：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="EnemyCombo">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>571</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FightButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>战斗！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UPAIButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上传AI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CompileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>编译</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>您现在的AI：</string>
     </property>
    </widget>
    <widget class="QLabel" name="AILabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TeamLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
